crates/types/src/config/cli_option_overrides.rs

```diff
@@ -74,10 +75,21 @@ pub struct CommonOptionCliOverride {
     #[clap(long, env = "RESTATE_CLUSTER_NAME", global = true)]
     pub cluster_name: Option<String>,
 
-    /// If true, then a new cluster is bootstrapped. This node *must* have an admin
-    /// role and a new nodes configuration will be created that includes this node.
-    #[clap(long, global = true)]
-    pub allow_bootstrap: Option<bool>,
+    /// Auto Cluster Provisioning
+    ///
+    /// If true, then this node is allowed to automatically provision as a new cluster.
+    /// This node *must* have an admin role and a new nodes configuration will be created that includes this node.
```

Comment:
I think it is no longer required that the node that provisions the cluster runs the admin role.